components/tidb_query/src/codec/mysql/decimal.rs

```diff
@@ -434,8 +434,9 @@ fn do_sub<'a>(mut lhs: &'a Decimal, mut rhs: &'a Decimal) -> Res<Decimal> {
     res
 }
 
+// TODO: add check for prec and frac_cnt
```

Comment:
Prefer to remove this comment and add an `assert(prec > frac_cnt)` (this TODO is meaningless because it's easy to fix it).